File: Mishka and Game.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int n;
    cin>>n;
    int m=0;
    int c=0;
    while(n--){
        int x;
        int y;
        cin>>x>>y;
        if(x>y){
            m = m+2;
        } else if(x<y){
            c = c+2;
        } else if(x==y){
            c= c+1;
            m= m+1;
        }
    }
    if(m>c){
        cout<<"Mishka \n";
    } else if(m<c){
        cout<<"Chris \n";
    } else if(m==c){
        cout<<"Friendship is magic!^^ \n";
    }
    return 0;
}

File: Mishka and Game.py
m = 0
c = 0

for _ in range(int(input())):
    x, y = [int(x) for x in input().split()]
    if x > y:
        m = m + 2
    elif x < y:
        c = c + 2
    elif x == y:
        m = m + 1
        c = c + 1
if m > c:
    print("Mishka")
elif m < c:
    print("Chris")
elif m == c:
    print("Friendship is magic!^^")
